project/Dependencies.scala

```diff
@@ -121,6 +121,7 @@ object Dependencies {
     // see https://github.com/GoogleCloudDataproc/spark-bigquery-connector/issues/36
     // Add the jar file to spark dependencies
     "com.google.cloud.spark" %% "spark-bigquery-with-dependencies" % Versions.sparkBigqueryWithDependencies % "provided" excludeAll (jacksonExclusions: _*),
+    "com.google.cloud.spark" % "spark-bigquery-connector-common" % Versions.sparkBigqueryConnectorCommon % "provided" excludeAll (jacksonExclusions: _*),
```

Comment:
What is this about ?

Reply:
This is a dependency used by spark bigquery. Without this, I don't get source code automatically while debugging.

Reply:
Good to know